### ISSUE TITLE
How many iters for training?

### ISSUE THREAD

**Issue by jamalknight** [karfly/learnable-triangulation-pytorch#92]
Hi there

When training (volumetric) there is an output in the terminal of the amount of iters done.

`420000 iters done. `

How many iters need to be completed for the training to be complete?

Are 'iters' the amount of images to be processed?

Thanks.

**Comment by karfly** [comment]
Hi, @jamalknight!
Can you share your loss plots?
To be honest, I don't remember the exact number of iterations, but it took about a week on a single NVIDIA P40 GPU.

**Comment by jamalknight** [comment]
Hi @karfly 
How do I share my loss plots?
(I am using a batch size of 1, so expecting my model to take longer than yours to train)

**Comment by karfly** [comment]
@jamalknight, just screenshot will be okay.

**Comment by jamalknight** [comment]
Hi @karfly 

Below are screengrabs of tensorboard logs:
(alternatively http://localhost:6006/#scalars)

![tensorboard_1](https://user-images.githubusercontent.com/38970401/87372887-f8b70b80-c5cb-11ea-881c-e83f7d15b5b0.png)

![tensorboard_2](https://user-images.githubusercontent.com/38970401/87372899-fe145600-c5cb-11ea-9e91-979d56b839e1.png)

![tensorboard_3](https://user-images.githubusercontent.com/38970401/87372906-0371a080-c5cc-11ea-8576-3738053c17ba.png)

![tensorboard_4](https://user-images.githubusercontent.com/38970401/87372915-08365480-c5cc-11ea-8e5a-d8d39d8c68c8.png)

**Comment by jamalknight** [comment]
Hi @karfly 

Any estimation when training will be done?

**Comment by jamalknight** [comment]
A week of training on my machine took approx 200 000 iters
4 weeks of training tool 825 000 iters
